### PR TITLE
Feature/server modal errors

### PR DESCRIPTION
**Pull request by emilyseibert**
# PR for Issue #154 and #278 .

## PR Status

**READY**

## Changes proposed in the pull request:
- General Server Errors message in modal
- Special alerts for game full and game has started
- Inline Server errors in modal

## Breaking changes?
- None

## Screenshot of feature:
image
image

### CODE REVIEW

File: src/cljs/wombats_web_client/components/cards/game.cljs [hunk 1]
```diff
@@ -45,15 +45,15 @@
   (let [game-state (get-game-state-str is-full is-playing)]
 
     [:div.arena-preview
-     (when game-state
+     (when false ;;game-state
```
Comment (oconn) [inline src/cljs/wombats_web_client/components/cards/game.cljs]: never gets executed?

File: src/cljs/wombats_web_client/components/cards/game.cljs [hunk 2]
```diff
@@ -45,15 +45,15 @@
...
        [:div.game-state-wrapper
         [:div.state-overlay]
         [:div.game-state game-state]])
      [:img {:src "/images/mini-arena.png"}]
      (when is-open
        [:div.countdown "Starts in "
         [countdown-timer start-time]])
-     (when is-joinable
+     (when true ;;is-joinable
```
Comment (oconn) [inline src/cljs/wombats_web_client/components/cards/game.cljs]: same

**Comment by emilyseibert** [comment]
dah fuck

File: src/cljs/wombats_web_client/components/modals/game-full-modal.cljs [hunk 3]
```diff
@@ -12,6 +11,7 @@
   (fn []
     [:div {:class "modal game-full-modal"}
      [:div.title "GAME FULL"]
-     [:div.desc full-msg]
+     [:div.desc game-full-error]
      [:div.action-buttons
-      [:button.simple-button {:on-click close-modal} "OKAY"]]]))
+      [:button {:class "close-button"
```
Comment (dehli) [inline src/cljs/wombats_web_client/components/modals/game-full-modal.cljs]: `:button.close-button`

File: src/cljs/wombats_web_client/components/modals/game-started-modal.cljs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+(ns wombats-web-client.components.modals.game-started-modal
+  (:require [re-frame.core :as re-frame]
+            [wombats-web-client.utils.errors :refer [game-started-error]]))
+
+(defn close-modal []
+  (re-frame/dispatch [:set-modal nil]))
+
+(defn game-started-modal []
+  (fn []
+    [:div {:class "modal game-started-modal"}
```
Comment (dehli) [inline src/cljs/wombats_web_client/components/modals/game-started-modal.cljs]: `:div.modal.game-started-modal`

File: src/cljs/wombats_web_client/components/modals/game-started-modal.cljs [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+     [:div.title "GAME HAS STARTED"]
+     [:div.desc game-started-error]
+     [:div.action-buttons
+      [:button {:class "close-button"
```
Comment (dehli) [inline src/cljs/wombats_web_client/components/modals/game-started-modal.cljs]: class

File: src/cljs/wombats_web_client/components/modals/join-wombat-modal.cljs [hunk 6]
```diff
@@ -107,7 +123,7 @@
                              :opacity "0.8"}}]
      [:img.selected-icon {:class (when selected "display") :src "/images/play.svg"}]
      [:img.wombat {:src (str "/images/wombat_" color-text "_right.png")
-            :onClick on-click-fn}]]))
+                   :onClick on-click-fn}]]))
```
Comment (dehli) [inline src/cljs/wombats_web_client/components/modals/join-wombat-modal.cljs]: on-click

**Comment by emilyseibert** [comment]
@oconn @dehli made updates

**Comment by dehli** [comment]
Looks good! Nice job. I think there's just one comment you forgot (the `on-click` change).

**Pull request merged**